src/GraphQLPlaygroundServiceProvider.php

```diff
@@ -30,13 +30,10 @@ public function boot()
             return;
         }
 
-        \Route::get(config('graphql-playground.route'), [
-            'middleware' => config('graphql-playground.middleware', ''),
-            'as' => 'graphql-playgroundcontroller',
```

Comment:
Is this deliberately removed?

In general, wouldn't it suffice to change just the `uses` to the class/method you added?

Reply:
Thanks for reviewing.

You are right. Probably changing only "uses" would suffice. It's more a personal preference - and, to be I didn't knew the syntax of the config array for the route action and didn't find documentation for it (didn't look for a long while though).
So I replaced it with the syntax familiar to me.

The only thing missing in the changed syntax is the `'as' => 'graphql-playgroundcontroller'`-part.
I did not see a reason to keep this. Am I missing something here?

Reply:
I do like the fluid syntax you used better as well. Pants down, i did not even test it. Depending all on you @jbbr ;)

Reply:
>  I did not see a reason to keep this. Am I missing something here?

It's the equivalent of naming a route.[ I've added it](https://github.com/mll-lab/laravel-graphql-playground/commit/20b41bae5f8f3c3e63ae3c4c3d8e9fd7c4b0f67a#diff-b72fdb5169f90ee8390eab8a1811a0a9R35) so from your application you can hook into a specific route from outside the package (it's a "good practice") and now you've removed it :-/